packages/BetterPhpDocParser/PhpDocParser/PhpDocFromTypeDeclarationDecorator.php

```diff
@@ -92,10 +92,9 @@ public function decorate(ClassMethod | Function_ | Closure | ArrowFunction $func
             return;
         }
 
-        $attributeGroup = $this->phpAttributeGroupFactory->createFromClass(
+        $functionLike->attrGroups[] = $this->phpAttributeGroupFactory->createFromClass(
             self::RETURN_TYPE_WILL_CHANGE_ATTRIBUTE
         );
-        $functionLike->attrGroups[] = $attributeGroup;
     }
```

Comment:
These snippets are now harder to read, as we assign somethign to array dim fetch. But it's unclear what is it. 
Before the named variable showed the context, and had less cognitive complexity. 

This should be kept as it is.

Reply:
ok,  I am closing it then.